docs/tool_jmap.md

```diff
@@ -69,4 +69,12 @@ The following tool limitations apply:
 - Displaying data from core dumps is not supported; use `jdmpview` instead.
 - Other options , such as `-F` (force a dump of an unresponsive process) can be accomplished using `kill -QUIT <pid>`.
 
+The tool uses the Attach API, and has the following limitations:
+
+- Displays information only for local processes that are owned by the current user, due to security considerations.

```

Comment:
This sentence (starting with "Displays information only for local processes...") actually already appears in this topic, in the previous section. There's more info there too so I'd delete this bullet point. The other topics are ok.